lib/stdlib/src/erl_stdlib_errors.erl

```diff
@@ -78,7 +78,7 @@ format_binary_error(decode_hex, [Subject], _) ->
         true ->
             [must_be_binary(Subject)]
     end;
-format_binary_error(decode_hex_char, [Subject], _) ->
+format_binary_error(unhex, [Subject], _) ->
     if
         is_integer(Subject),
```

Comment:
I think we can remove the `if` here and always return `[range]`. Or maybe even better to return `[<<"not a valid hex">>]`.